addons/units/CfgVehicles_PublicUnits.hpp

```diff
@@ -274,6 +334,10 @@ class CLASS(Unit_I_PilotHeli): CLASS(Unit_Combat_RS_BS_GP_BB) {
             speechPlural[] = {"veh_infantry_pilot_p"};
         };
     };
+
+    CBA_facewearList[] = {
+        "", 1
+    };
```

Comment:
If the facewearList only has one entry anyway, isn't it a bit redundant to add it?

Reply:
No, read documentation: https://github.com/CBATeam/CBA_A3/blob/ac776664cbc160744f58c539a065ffefe3aa4cb6/addons/common/fnc_randomizeFacewear.sqf#L9

Reply:
Ah, didn't see the doc yet.
Thought if you just had one facewear applied in linkedItems, you wouldn't need the facewearList, similar to the headgear.

All looks good to me then.

Reply:
`linkedItems` is just the default. If you inherit a unit with randomization you might still have randomization, this disables it 100%.